spec.html

```diff
@@ -15457,6 +15466,19 @@ <h1>Static Semantics: Early Errors</h1>
         <emu-note>
           <p>This production exists so that |ObjectLiteral| can serve as a cover grammar for |ObjectAssignmentPattern|. It cannot occur in an actual object initializer.</p>
         </emu-note>
+        <emu-grammar>
+          ObjectLiteral : `{` PropertyDefinitionList `}`
+
+          ObjectLiteral : `{` PropertyDefinitionList `,` `}`
+        </emu-grammar>
+        <ul>
+          <li>
+            It is a Syntax Error if PropertyNameList of |PropertyDefinitionList| contains any duplicate entries for *"__proto__"* and at least two of those entries were obtained from productions of the form <emu-grammar>PropertyDefinition : PropertyName `:` AssignmentExpression</emu-grammar>. This rule is not applied if this |PropertyDefinition| is contained within a |Script| which is being evaluated for <emu-xref href="#sec-json.parse">JSON.parse</emu-xref> (see step <emu-xref href="#step-json-parse-parse"></emu-xref> of <emu-xref href="#sec-json.parse">JSON.parse</emu-xref>).
```

Comment:
```suggestion
            It is a Syntax Error if PropertyNameList of |PropertyDefinitionList| contains any duplicate entries for *"__proto__"* and at least two of those entries were obtained from productions of the form <emu-grammar>PropertyDefinition : PropertyName `:` AssignmentExpression</emu-grammar>. This rule is not applied if this |PropertyDefinition| is contained within a |Script| that is being parsed for JSON.parse (see step <emu-xref href="#step-json-parse-parse"></emu-xref> of <emu-xref href="#sec-json.parse">JSON.parse</emu-xref>).
```
1. s/which/that/
2. s/evaluated/parsed/ because early error rules are applied (or not) at parse-time.
3. We don't need 2 links to JSON.parse in quick succession. (I removed the linking from the first one to match line 15577, but you could go the other way.)